### PR TITLE
Update to iOS 9

### PR DESCRIPTION
**Pull request by sendhil**
This PR flips the iOS9 switch and also disabled the deprecation warnings for now because there were almost 300 of them. Fixing and testing all of them without a huge merge nightmare was unfeasible so  we're going to slowly patch the deprecation warnings over the next season. That being said we had to patch a few of them now because disabling the warnings didn't disable warnings for Swift and having 30 or so warnings in a project is undesirable because it runs the risk of hiding real legit warnings.

### CODE REVIEW

File: WordPress/Classes/Utility/WPWebViewController.m [hunk 1]
```diff
@@ -454,7 +444,16 @@ - (NSURLRequest *)newRequestForWebsite
         return [WPURLRequest requestWithURL:self.url userAgent:userAgent];
     }
     
-    NSURL *loginURL = self.wpLoginURL ?: [[NSURL alloc] initWithScheme:self.url.scheme host:self.url.host path:@"/wp-login.php"];
+    NSURL *loginURL = self.wpLoginURL;
+    
+    if (!loginURL) {
+        // Thank you, iOS 9, everything is more compact and pretty, now.
```
Comment (sendhil) [inline WordPress/Classes/Utility/WPWebViewController.m]: :dancer:

**Comment by kurzee** [comment]
I took a look-through the commits and made minor notes on 3b4790a2d497cbf7c9e3c738f50b5602576c6523

File: WordPress/Classes/Extensions/UIAlertController+Helpers.swift [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        //
+        guard let rootViewController = UIApplication.sharedApplication().delegate?.window??.rootViewController else {
+            print("Error loading the rootViewController")
+            return
+        }
+        
+        var leafViewController = rootViewController
+        while leafViewController.presentedViewController != nil {
+            leafViewController = leafViewController.presentedViewController!
+        }
```
Comment (kurzee) [inline WordPress/Classes/Extensions/UIAlertController+Helpers.swift]: @jleandroperez do you think this could fail with other view controller hierarchies? In the past this has always been fairly complicated given navigation controllers and additional presented view controllers. This might not be guaranteed to work in upcoming iOS versions, but I suppose there may not be a guaranteed way at all.

I tested this with different view controller/navigation controller/presented controller setups and it worked in each. There may be some extra logic within `UIAlertController` taking care of this for us.

**Comment by sendhil** [comment]
Going to merge this for now as per my discussion with @kurzee so we can crank out a beta and we can keep on testing the issues raised throughout the course of the beta.

File: WordPress/Classes/Extensions/UIAlertController+Helpers.swift (hunk 2 above)
Comment (jleandroperez) [inline WordPress/Classes/Extensions/UIAlertController+Helpers.swift]: @kurzee i'm certainly not happy with that loop. It _should be_ safe, unless something huge changes.

Personally, i'd love to just remove that snippet altogether. Any suggestions would be more than welcome sir!

By the way, i've just realized that we should be grabbing the *key window* instead. Will submit that fix in another PR!
